src/components/HeaderNavMenu.jsx

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
+import firebase from 'firebase';
 import { makeStyles, Link as MuiLink, Grid } from '@material-ui/core';
-import LoginButton from './LoginButton';
+import Button from '@material-ui/core/Button';
+import ProfileMenu from './ProfileDropdown';
 
 const useStyles = makeStyles((theme) => ({
   menu: {
-    alignItems: 'center',
+    alignItems: 'right',
```

Comment:
This is an invalid value so it makes the menu items not aligned anywhere.

Reply:
Ah ok so should I leave it as `center` or remove this line?